calypso/service.go

```diff
@@ -342,6 +353,11 @@ func (s *Service) ReshareLTS(req *ReshareLTS) (*ReshareLTSReply, error) {
 
 		// NOTE: the roster stored in ByzCoin must have myself.
 		tree := roster.GenerateNaryTreeWithRoot(len(roster.List), s.ServerIdentity())
+		if tree == nil {
+			return nil, xerrors.New("failed to generate tree " +
```

Comment:
As we only support go 1.13 and go 1.14, you can use `errors.New` here, also below, where `fmt.Errorf` can be used instead of `xerrors.Errorf`.

Reply:
Since the rest of the code uses `xerrors`, I'd rather use that for conformity. We can open a new PR that updates all errors if needed.